InnerEye/ML/model_training.py

```diff
@@ -290,3 +280,25 @@ def model_train(config: ModelConfigBase,
         resource_monitor.kill()
 
     return model_training_results
+
+
+def aggregate_and_create_subject_metrics_file(outputs_folder: Path) -> None:
+    """
+    This functions takes all the subject metrics file written by each GPU (one file per GPU) and aggregates them into
+    one single metrics file. Results is saved in config.outputs_folder / mode.value / SUBJECT_METRICS_FILE_NAME.
+    This is done for the metrics files for training and for validation data separately.
+    :param config: model config
+    """
+    for mode in [ModelExecutionMode.TRAIN, ModelExecutionMode.VAL]:
+        temp_files = (outputs_folder / mode.value).rglob(SUBJECT_OUTPUT_PER_RANK_PREFIX + "*")
+        result_file = outputs_folder / mode.value / SUBJECT_METRICS_FILE_NAME
+        result_file = result_file.open("a")
+        for i, file in enumerate(temp_files):
+            temp_file_contents = file.read_text()
+            if i == 0:
+                # Copy the first file as-is, including the first line with the column headers
+                result_file.write(temp_file_contents)
+            else:
+                # For all files but the first one, cut off the header line.
+                result_file.write("\n" + os.linesep.join(temp_file_contents.splitlines()[1:]))
```

Comment:
inconsistent use of `'\n'` and `os.linesep`

Reply:
good catch!